From 3c4c655477441d89ff0c6488703a6831755274b2 Mon Sep 17 00:00:00 2001
From: Pavol Juhas <pavol.juhas@gmail.com>
Date: Fri, 2 Feb 2018 16:16:54 -0500
Subject: REL: use git commit date for version metadata

Use commit time stamp as in git-archive and versioneer.
---
 .gitarchive.cfg | 4 ++--
 setup.py        | 2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/.gitarchive.cfg b/.gitarchive.cfg
index 43b349f..95e1448 100644
--- a/.gitarchive.cfg
+++ b/.gitarchive.cfg
@@ -1,5 +1,5 @@
 [DEFAULT]
 commit = $Format:%H$
-date = $Format:%ai$
-timestamp = $Format:%at$
+date = $Format:%ci$
+timestamp = $Format:%ct$
 refnames = $Format:%D$
diff --git a/setup.py b/setup.py
index fbd0294..9218f5a 100755
--- a/setup.py
+++ b/setup.py
@@ -26,7 +26,7 @@ def gitinfo():
     kw = dict(stdout=PIPE, cwd=MYDIR, universal_newlines=True)
     proc = Popen(['git', 'describe', '--match=v[[:digit:]]*'], **kw)
     desc = proc.stdout.read()
-    proc = Popen(['git', 'log', '-1', '--format=%H %at %ai'], **kw)
+    proc = Popen(['git', 'log', '-1', '--format=%H %ct %ci'], **kw)
     glog = proc.stdout.read()
     rv = {}
     rv['version'] = '.post'.join(desc.strip().split('-')[:2]).lstrip('v')
-- 
2.16.0

